ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -83,4 +58,89 @@ public extension UIColor {
             }
         }
     }
+
+    /// Creates a UIColor from a single 32-bit integer value.
+    ///
+    /// - Parameter hexValue: Integer value, generally represented as hexadecimal (e.g. `0x0086F0`), to use to initialize this color.
+    ///     Must be formatted as ARGB. Note: we will not utilize the alpha channel.
+    convenience init(colorValue: ColorValue) {
+        self.init(
+            red: CGFloat((colorValue & 0xFF0000) >> 16) / 255.0,
+            green: CGFloat((colorValue & 0x00FF00) >> 8) / 255.0,
+            blue: CGFloat((colorValue & 0x0000FF) >> 0) / 255.0,
+            alpha: 1.0)
+    }
+
+    /// Creates a dynamic color object that returns the appropriate color value based on the current
+    /// rendering context.
+    ///
+    /// - Parameter colorSet: The set of color values that may be applied based on the current context.
+    convenience init(colorSet: ColorSet) {
+        self.init { traits -> UIColor in
+            let colorValue = colorSet.value(colorScheme: (traits.userInterfaceStyle == .dark ? .dark : .light),
+                                            contrast: traits.accessibilityContrast == .high ? .increased : .standard,
+                                            isElevated: traits.userInterfaceLevel == .elevated ? true : false)
```

Comment:
just `traits.userInterfaceLevel == .elevated` instead of the ternary operator?

Reply:
Sure! I was just going for symmetry :)